src/main/java/jenkins/branch/DeadBranchStrategy.java

```diff
@@ -45,7 +45,7 @@
      */
     private transient MultiBranchProject<?, ?> owner;
 
-    public abstract <P extends AbstractProject<P, R> & TopLevelItem, R extends AbstractBuild<P, R>>
+    public abstract <P extends Job<P, R> & TopLevelItem, R extends Run<P, R>>
     void runDeadBranchCleanup(TaskListener listener, Map<String, P> deadBranches) throws IOException,
             InterruptedException;
```

Comment:
Workflows are not inheriting from `AbstractProject`. You'll see similar changes for workflow compatibility all over the place.